src/components/button.js

```diff
@@ -0,0 +1,17 @@
+import ButtonCSS from './Button.module.css';
```

Comment:
This file is still named button instead of Button. Which will cause problems with other's computers like mine
error [image](https://user-images.githubusercontent.com/14869087/148965129-33675738-7327-4e00-b9af-b8360145261c.png)

When dealing with case sensitive renames, you need to rename it using the terminal. [tutorial](https://sebastiandedeyne.com/how-to-do-a-case-sensitive-file-rename-in-git-on-macos/)


Also, a common pattern to name all component files in CamelCase. So you'd also need to rename the other component files (eg Day.js, Cell.js, etc...)